electron_src/main.js

```diff
@@ -66,7 +67,64 @@ function createWindow() {
   if (windowState.maximized) {
     mainWindow.maximize();
   }
-  mainWindow.show();
+  mainWindow.on('ready-to-show', () => {
+    mainWindow.webContents.setZoomLevel(windowState.zoomLevel);
```

Comment:
exception when start electron process:

> Uncaught Exception:
> TypeError: Error processing argument at index 0, conversion failure from undefined
> at BrowserWindow

Reply:
There's already a default zoomLevel defined in store.

Reply:
migration problem. former windowState data has no zoomlevel field.